pages/organizer/dashboard.jsx

```diff
@@ -1,24 +1,75 @@
 import React from "react";
-import { Flex, Heading } from '@chakra-ui/react'
+import { Flex, Heading, VStack, Text } from '@chakra-ui/react'
 import Event from "../../components/Event/Event"
 import StallRequest from "../../components/Event/StallRequest";
+import { signOut, getSession } from "next-auth/react";
 
-const OrganizerDashboard = () => {
-  return <Flex h="100vh" direction="column" align="center">
+
+const OrganizerDashboard = ({session, events, stallRequests}) => {
+  return <Flex  direction="column" align="center">
     <Heading mt={10} as='h3' size='lg'>
     My Events
   </Heading>
   <Flex align="center" w="50%" direction="column" mt={8}>
-    <Event/>
+    {events.events.map((event)=>{
+      return <Event key={event._id} name={event.name} date={event.date} venue={event.venue} imageUrl={event.images[0]}/>
+    })}
```

Comment:
Why pass down the specific properties instead of everything?

Reply:
And then destructuring what i need in the event component. Ill clean it up